src/Farmhand.js

```diff
@@ -1117,10 +1240,16 @@ export default class Farmhand extends Component {
       viewTitle,
     } = this
 
+    const blockInput =
```

Comment:
not a must change, but this could be a getter instead. though if it will never applicable anywhere else it doesn't matter.